### PR TITLE
Fix the Offset, Zoffset used by TintedCellsLayer

### PR DESCRIPTION
**Pull request by dnqbob**
new colo3

### CODE REVIEW

File: OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs [hunk 1]
```diff
@@ -46,6 +46,9 @@ public class TintedCellsLayerInfo : TraitInfo
 		[Desc("Z offset of the visualization.")]
 		public readonly int ZOffset = 512;
```
Comment (MustaphaTR) [inline OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: We already had a ZOffset up here. How is the two any different other than now allowing X and Y too? Is this value used somewhere else? If not it should be removed.

Comment (dnqbob) [reply OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: Offset (xyz) is a quite different thing compared to ZOffset(or Zsorting). Offset will actually affect the position of the image, while ZOffset decides put which image to the top when many images takes the same position.

In the gif I showed above,  tinted cell (considered as an image) has the same position as unit, but its ZOffset set to very low so it make sure unit always on the top of tinted cell.

Set the same value into Z axis of Offset will result in the entire cells move downward.

At last, as long as we initialise the ZOffset in 'TintedCell' class, the upper caller will use it to adjust the image.

Comment (dnqbob) [reply OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: It's used here
Screenshot_20241103_141543_org_mozilla_firefox_HomeActivity_edit_10925252420883

Comment (dnqbob) [reply OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: and at `WorldRenderer` when it is drawing, the zoffset is called

 to adjust which image on the top.

Comment (dnqbob) [reply OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: so we should not touch zoffset in renderable object, just initializing the interface is enough.

**Comment by dnqbob** [comment]
Also set the default value of Zoffset and offset to 0.

File: OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs (hunk 1 above)
Comment (MustaphaTR) [reply OpenRA.Mods.AS/Traits/World/TintedCellsLayer.cs]: Yeah, sorry. I had just woken up. Only remembered about the difference while having breakfast.

**Pull request merged**